examples/src/keyrings/set_encryption_algorithm_example.py

```diff
@@ -0,0 +1,151 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+This example demonstrates how to set an encryption algorithm while using the Raw AES Keyring
+in the AWS Encryption SDK.
+
+The encryption algorithm used in the encrypt() method is the algorithm used to protect your
+data using the data key. By setting this algorithm, you can configure the algorithm used
+to encrypt and decrypt your data.
+
+Encryption algorithms can be set in a similar manner in other keyrings as well. However,
+please make sure that you're using a logical encryption algorithm that is compatible with your
+keyring. For more information on encryption algorithms supported by the AWS Encryption SDK, see
+https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/supported-algorithms.html
+
+The AES wrapping algorithm (AesWrappingAlg.ALG_AES256_GCM_IV12_TAG16) protects your data key using
+the user-provided wrapping key. The encryption algorithm used in the encrypt() method is the
+algorithm used to protect your data using the data key. This example demonstrates setting the
+latter, which is the encryption algorithm for protecting your data. When the commitment policy is
+REQUIRE_ENCRYPT_REQUIRE_DECRYPT, the default algorithm used in the encrypt method is
+AES_256_GCM_HKDF_SHA512_COMMIT_KEY_ECDSA_P384, which is a committing and signing algorithm.
+Signature verification is extremely useful to ensure the integrity of a digital message as it
+goes between systems. However, signature verification adds a significant performance cost on
+decryption. If the users encrypting data and the users decrypting data are equally trusted, we can
+consider using an algorithm suite that does not include signing. This example sets the encryption
+algorithm as AES_256_GCM_HKDF_SHA512_COMMIT_KEY which is a committing but non-signing algorithm.
+For more information on digital signatures, see
+https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/concepts.html#digital-sigs
```

Comment:
```suggestion
The AES wrapping algorithm (AesWrappingAlg.ALG_AES256_GCM_IV12_TAG16) protects your data key using
the user-provided wrapping key. In contrast, the encryption algorithm used in the encrypt() method is the
algorithm used to protect your data using the data key. This example demonstrates setting the
latter, which is the encryption algorithm for protecting your data. When the commitment policy is
REQUIRE_ENCRYPT_REQUIRE_DECRYPT, the default algorithm used in the encrypt method is
AES_256_GCM_HKDF_SHA512_COMMIT_KEY_ECDSA_P384, which is a committing and signing algorithm.
Signature verification ensures the integrity of a digital message as it
goes between systems. However, signature verification adds a significant performance cost to
encrypting or decryption. If encryptors and decryptors are equally trusted, we can
consider using an algorithm suite that does not include signing. This example sets the encryption
algorithm as AES_256_GCM_HKDF_SHA512_COMMIT_KEY, which is a committing but non-signing algorithm.
For more information on digital signatures, see
https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/concepts.html#digital-sigs
```

Just touching some stuff up, feel free to push back

Reply:
Thanks, made some minor changes to this and added